.github/workflows/esti.yaml

```diff
@@ -721,6 +721,68 @@ jobs:
         working-directory: test/spark
         run: docker-compose logs --tail=1000 lakefs
 
+  run-system-aws-s3-kv:
+    name: Run latest lakeFS app on AWS S3 KV
+    needs: [gen-code, deploy-image]
+    runs-on: ubuntu-20.04
+    env:
+      TAG: ${{ needs.deploy-image.outputs.tag }}
+      # Setting Account_ID as a secret as a way to avoid specifying it here
+      REPO: ${{ secrets.AWS_ACCOUNT_ID }}.dkr.ecr.us-east-1.amazonaws.com
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    steps:
+      - name: Check-out code
+        uses: actions/checkout@v2
+
+      - name: Generate uniquifying value
+        id: unique
+        run:  echo "::set-output name=value::$RANDOM"
+
+      - name: Start lakeFS with S3 tests KV
+        uses: ./.github/actions/bootstrap-test-lakefs
+        with:
+          compose-file: esti/ops/docker-compose.yaml
+          compose-flags: "--quiet-pull --exit-code-from=esti"
+        env:
+          LAKEFS_BLOCKSTORE_TYPE: s3
+          LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID: ${{ secrets.ESTI_AWS_ACCESS_KEY_ID }}
+          LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY: ${{ secrets.ESTI_AWS_SECRET_ACCESS_KEY }}
+          LAKEFS_DATABASE_KV_ENABLED: true
```

Comment:
Dumb question but learning from graveler migration: How would we know that the KV was actually used?

Reply:
It includes the log warning that we currently "USING KV EXPERIMENTAL FLAG!!! USE AT YOUR OWN RISK!!!"